common/app/views/fragments/amp/analytics.scala.html

```diff
@@ -1,7 +1,7 @@
 @import play.api.Mode
 @import model.Tag
 @import play.api.libs.json.Json
-@(content: model.Content)(implicit request: RequestHeader, context: model.ApplicationContext)
+@(content: model.Content, page: model.Page)(implicit request: RequestHeader, context: model.ApplicationContext)
```

Comment:
You could possibly pass a `ContentPage` param here and then only need one param, rather than two.
Assuming that the page passed in is always a content page.